### DISCUSSION TITLE
What are the high-lighted rectangle in the graph?

### DISCUSSION THREAD

**Discussion by hz7451** [kernc/backtesting.py#543] [General]
![image](https://user-images.githubusercontent.com/58805643/143605880-0dbea35c-8ca5-4626-a8d1-c55cdc7e877f.png)

**Comment by kernc** [comment]
> What are the high-lighted rectangles in the graph?

Candles for a higher timeframe. See [`Backtest.plot(superimpose=)`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest.plot).

**Reply by Hawar-Dzaee** [reply]
you mean higher as this is in the documents "If superimpose is True, superimpose larger-timeframe candlesticks over the original candlestick chart". I don't think a lower timeframe is possible, ( if you have CSV file of hourly time frame, while it can be resampled to any higher time frame, but it can not be broken down lower timeframes).
